### DISCUSSION TITLE
Home Mini - Gas - Current Consumption

### DISCUSSION THREAD

**Discussion by liambresnahan** [BottlecapDave/HomeAssistant-OctopusEnergy#247] [Q&A]
Hi All,

Looking for some guidance. I have the Octopus Home Mini and have been trying to use my gas `current_consumption` in my energy dashboard but am not able to use the entity. Is this a known issue? The only way I seem to be able to add an entity is using the `previous_accumulative_consumption`. 

Is this a known issue? Or am I doing something wrong?

Cheers!

**Comment by BottlecapDave** [comment]
Hello and sorry that you're having issues. Have you followed the instructions as laid out in https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/energy_dashboard.md#current-consumption. If so, what error are you receiving?

**Reply by BottlecapDave** [reply]
Additionally, is the sensor available? Do you have gas available under "today" for "day" within the Octopus Energy app? If not, this will be why it's not available in HA as the integration only retrieves the same data that the Octopus Energy app exposes. However I'm sure when I first setup the Octopus Mini it took a few days for gas to start appearing in the app (and HA). Once it appears in the app, if you reload the integration (or restart your instance of HA) then it should appear.

**Reply by liambresnahan** [reply]
Thanks so much for getting back to me. That’s where I got the instructions on adding this, yes. I don’t get an error on this, when entering the entity into the Gas Usage box I don't get it actually show up in the search at all

**Reply by liambresnahan** [reply]
> Additionally, is the sensor available? Do you have gas available under "today" for "day" within the Octopus Energy app? If not, this will be why it's not available in HA as the integration only retrieves the same data that the Octopus Energy app exposes. However I'm sure when I first setup the Octopus Mini it took a few days for gas to start appearing in the app (and HA). Once it appears in the app, if you reload the integration (or restart your instance of HA) then it should appear.

I can see the entity has data, but for some reason it isn't coming up in the search box of gas usage. Perhaps I have to wait a bit as I only got the home mini today. So I'll give it a few days. Thanks again for getting back to me
